Added function to load model from XGBoost json file

simonlet opened the pull request.

simonlet committed:
Added function to load model from XGBoost json file

sbuschjaeger reviewed (commented):
Path(file_path).stem ist dann mit der Endung *.json? Das wäre blöd
Bitte noch sinnvolle asserts / exception einbauen wenn a) file_path die falsche Endung hat und b) der Zugriff auf das dictionary failed. 

Gibt es eine Möglichkeit die Anzahl der Samples pro Node aus xgboost zu bekommen?

simonlet commented:
1. `.stem` entfernt die Dateiendung, also ohne `.json`.
2. Mach ich.
3. Bestimmt, ich schau gerade mal.

simonlet committed:
Added error handling to model_from_xgb_file
Also base_score gets added to leaf predictions and num_features is now parsed to int
